doc/nrf/drivers/sensor_stub.rst

```diff
@@ -27,12 +27,13 @@ You can define instances of the sensor stub on Devicethree like this:
         };
 
 The most important setting here is the ``generator`` property.
-The driver would require the user to implement following functions:
-- ``int <generator>_init(const struct device *dev)`` - initialize the driver
-- ``int <generator>_get(const struct device *dev, enum sensor_channel ch, struct sensor_value *val)`` - get the measure value
-- ``int <generator>_fetch(const struct device *dev, enum sensor_channel ch)`` - fetch (prepare) the measured value
+The driver requires the following functions to be implemented:
 
-The functions has to have global linkage - they would be connected directly in the sensor stub driver.
+- ``int <generator>_init(const struct device *dev)`` - This function initializes the driver.
+- ``int <generator>_get(const struct device *dev, enum sensor_channel ch, struct sensor_value *val)`` - This function gets the measured value.
+- ``int <generator>_fetch(const struct device *dev, enum sensor_channel ch)`` - This function fetches (prepares) the measured value.
+
+The functions must have global linkage and must be connected directly to the sensor stub driver.
```

Comment:
The meaning of the sentence there is changed - it seems like it is the user responsibility to connect them to the sensor driver. The real meaning was that sensor would link to them itself.